x/stake/types/errors.go

```diff
@@ -117,6 +126,10 @@ func ErrNotMature(codespace sdk.CodespaceType, operation, descriptor string, got
 	return sdk.NewError(codespace, CodeUnauthorized, msg)
 }
 
+func ErrBadUnbondingDelegationAddr(codespace sdk.CodespaceType) sdk.Error {
```

Comment:
I see no difference between this and ErrBadDelegationAddr err messages.

Reply:
++

Reply:
True that, will amend it.

Reply:
Addressed - @melekes please review